frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -17,22 +17,44 @@
 
 //! The unsigned phase implementation.
 
-use crate::*;
-use frame_support::dispatch::DispatchResult;
+use crate::{
```

Comment:
would rather collapse this for consistency.

Reply:
a192ba3a0